models/article-model.js

```diff
@@ -48,3 +48,31 @@ exports.fetchArticles = (sort_by = "created_at", order = "DESC", next) => {
     return results.rows;
   });
 };
+
+exports.editVotes = (articleId, updateVote) => {
+  const queryValues = [updateVote.inc_votes, articleId];
+  console.log(queryValues);
+
+  const query =
+    "UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *;";
+
+  console.log(updateVote);
+  const checkifIdExists = "SELECT * FROM articles WHERE article_id = $1;";
+
+  if (Object.keys(updateVote).length === 0) {
+    return Promise.reject({ status: 400, message: "Bad Request" });
+  }
+  if(isNaN(updateVote.inc_votes)){
+    return Promise.reject({ status: 400, message: "Bad Request" });
+  }
```

Comment:
It's not technically wrong to check these manually, but it might be worth seeing what happens if you remove one of these and seeing if psql gives you an error back itself. If votes is not a number will the query fail? It could give you a code back that you could catch and use instead.